CONTRIBUTING.md

```diff
@@ -174,14 +176,19 @@ $ cd build                                                                # ente
 # -DCMAKE_CXX11_ABI set to ON or OFF depending on the ABI version you want, defaults to ON. When turned ON, ABI compability for C++11 is used. When OFF, pre-C++11 ABI compability is used.
 $ cmake .. -DCMAKE_INSTALL_PREFIX=$CONDA_PREFIX -DCMAKE_CXX11_ABI=ON      # configure cmake ...
 
+$ make nvstrings                                                          # compile libnvstrings dependency for libcudf
 $ make -j                                                                 # compile the libraries librmm.so, libcudf.so ... '-j' will start a parallel job using the number of physical cores available on your system
```

Comment:
Running `make -j` here should build the libnvstrings libraries, so you shouldn't need to run `make nvstrings` above.

Reply:
I agree. If we just do `make` it causes the linking issue, hence I had added the step prior.